syntax = "proto2";

option optimize_for = LITE_RUNTIME;

option java_package = "ProtoMessages";

option java_outer_classname = "MessageResponseRoundPlayerBangBase";

message MessageResponseRoundPlayerBang {
    required int64 playerId = 1 [default = 0];
    required int32 playerPosition = 2 [default = 0];
    required int64 killedPlayerId = 3 [default = 0];
    required int32 killedPlayerPosition = 4 [default = 0];
    required int32 winAmount = 5 [default = 0];
}
